api/v1/tetragon/tetragon.proto

```diff
@@ -546,6 +546,8 @@ message ProcessLsm {
     KprobeAction action = 8;
     // Tags of the Tracing Policy to categorize the event.
     repeated string tags = 9;
+    // IMA file hash. Format algorithm:value.
+    string ima_hash = 11;
```

Comment:
Is there a reason for not introducing a separate struct with the two fields?

E.g., 

```
message ImaFileHash {
    string (?) algorithm
    string (?) value
}
```

Question: Is the value of the hash encoded in  a string or is it bytes?

Reply:
I took the default representation of IMA hashes from here: 

```
sudo head /sys/kernel/security/integrity/ima/ascii_runtime_measurements
10 bfc19aede7d85629786e0d59148778666c755db9 ima-ng sha256:70b6c3ed60645b8c33e4ccfc0af769960ace357b0d461a0d812000f5e1aabbc8 boot_aggregate
10 0b5a1ee5112b581fc74040146ce5bdb607c1e1ac ima-ng sha256:d9d1775d643f6f70a1a6f646dfe023052765f558a167ec58951ad2f1013b3e46 /init
10 fbc981e4b231cc9b0677302e85761ad99b628c6f ima-ng sha256:f5f25da321f9341462205d8fd6ec918bba89ad5ea5dc9f6d875d5174d1936422 /usr/bin/sh
```

Also, docker image ID is a hash, stored the same way:

```
docker images --no-trunc --quiet $IMAGE
sha256:90a85ddf9c2540f149a2de34c5d1e3d88f566bbed754cc2ebb56c439d8ca7f09
sha256:30e6fd08ea57517b0c3325ccff2d3dd351959f082a95c5a14a49694aab647a30
sha256:caa045052df8cfc98fe60290a5d99c9995c75811d4d2f6df1cf062549d1cbbc8
```

> Question: Is the value of the hash encoded in a string or is it bytes?

Hash value here is just a hex string, no encoding is here such as base64, etc.

Reply:
I see, thanks! OK if it's the established way of doing things, let's leave it as is.